cl/phase1/forkchoice/forkchoice.go

```diff
@@ -48,9 +48,13 @@ type ForkChoiceStore struct {
 	latestMessages   map[uint64]*LatestMessage
 	anchorPublicKeys []byte
 	// We keep track of them so that we can forkchoice with EL.
-	eth2Roots             *lru.Cache[libcommon.Hash, libcommon.Hash] // ETH2 root -> ETH1 hash
-	preverifiedValidators *lru.Cache[libcommon.Hash, uint64]
-	mu                    sync.Mutex
+	eth2Roots *lru.Cache[libcommon.Hash, libcommon.Hash] // ETH2 root -> ETH1 hash
+	// preverifid sizes
```

Comment:
why do we need three different caches for these ?

arent' these hashes all supposed to be the same? maybe one cache is enough? 
libcommon.Hash -> [3]uint64 ?